bifrost/convert.go

```diff
@@ -87,8 +87,10 @@ func (c *DropletToImageConverter) Convert(request cf.DesireLRPRequest) (opi.LRP,
 
 	return opi.LRP{
 		AppName:         vcap.AppName,
+		AppGuid:         vcap.AppID,
```

Comment:
One question I had about this is how come sometimes we are reaching into the `VCAP_APPLICATION` environment variable JSON to get app metadata and sometimes we are receiving it directly from CAPI during the DesireLRP call?

I believe modern Cloud Controllers are including a `process_type` key in their `VCAP_APPLICATION` env, so I could technically get this from there.